recipes/openmm/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DBUILD_TESTING=OFF -DCMAKE_BUILD_TYPE=Release"
+
+if [[ "$target_platform" == linux* ]]; then
+    # CFLAGS
+    # JRG: Had to add -ldl to prevent linking errors (dlopen, etc)
+    MINIMAL_CFLAGS+=" -g -O3 -ldl"
+    CFLAGS+=" $MINIMAL_CFLAGS"
+    CXXFLAGS+=" $MINIMAL_CFLAGS"
+    LDFLAGS+=" $LDPATHFLAGS"
+
+    # Use GCC
+    CMAKE_FLAGS+=" -DCMAKE_C_COMPILER=$CC -DCMAKE_CXX_COMPILER=$CXX"
+
+    # CUDA_HOME is defined by nvcc metapackage
+    CMAKE_FLAGS+=" -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME}"
+    # From: https://github.com/floydhub/dl-docker/issues/59
+    CMAKE_FLAGS+=" -DCMAKE_LIBRARY_PATH=${CUDA_HOME}/lib64/stubs"
+    # CUDA tests won't build, disable for now
+    # See https://github.com/openmm/openmm/issues/2258#issuecomment-462223634
+    CMAKE_FLAGS+=" -DOPENMM_BUILD_CUDA_TESTS=OFF"
+    # CUDA OpenCL
+    CMAKE_FLAGS+=" -DOPENCL_INCLUDE_DIR=${CUDA_HOME}/include/"
+    CMAKE_FLAGS+=" -DOPENCL_LIBRARY=${CUDA_HOME}/lib64/libOpenCL.so"
```

Comment:
Does this need to be CUDA's OpenCL? You should use an ICD loader. See PyOpenCL recipe for the ICD loader library names.